### DISCUSSION TITLE
Add support for testing with Jest

### DISCUSSION THREAD

**Discussion by santiagopuentep** [meteor/meteor#12498] [Feature Requests]
Currently it seems impossible to test `Meteor` using `Jest`, which is by far the most popular testing solution. I think this is one of the many problems of the very old Meteor build system. `Vite + Jest`?

Check out this discussion about trying to use Jest with Meteor (which failed).

https://github.com/VulcanJS/Vulcan/issues/2270

**Comment by jamauro** [comment]
Also interested in using Jest
